department-of-veterans-affairs/abd-vro
# Workaround for Docker.com rate limit

## What was the problem?


Address Docker.com rate limit. Details in #932. 
We don't need to push/publish these 3rd-party images to GHCR as part of our GH Actions. A 3rd-party image only needs to be pushed once per version of the image.

## How does this fix it?[^1]


1.  Remove these 3rd-party images from being published by GH Actions.
2. Added a script `add-3rd-party-image.sh` for DevOps to push the 3rd-party image to GHCR once
3. Update Helm configurations to pull that image (instead of Docker.com images) for LHDI deploys
4. Update [SecRel GH Action workflow](https://github.com/department-of-veterans-affairs/abd-vro/blob/2e7d3cc5b7363f290ae49a9088e03a011440f5a0/.github/workflows/secrel4prs.yml#L57-L65) to include these images b/c SecRel wants to scan these 3rd-party images. .

## How to test this PR

```
scripts/add-3rd-party-image.sh rabbitmq 3
scripts/add-3rd-party-image.sh redis 7
```

Deploy to LHDI.

[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
